fvm/bootstrap.go

```diff
@@ -235,6 +323,52 @@ transaction(amount: UFix64) {
 }
 `
 
+const setupFeesTransactionTemplate = `
+import FlowStorageFees, FlowServiceAccount from 0x%s
+
+transaction(accountCreationFee: UFix64, minimumStorageReservation: UFix64) {
+    prepare(service: AuthAccount) {
+        let serviceAdmin = service.borrow<&FlowServiceAccount.Administrator>(from: /storage/flowServiceAdmin)
+            ?? panic("Could not borrow reference to the flow service admin!");
+
+        let storageAdmin = service.borrow<&FlowStorageFees.Administrator>(from: /storage/storageFeesAdmin)
+            ?? panic("Could not borrow reference to the flow storage fees admin!");
+
+        serviceAdmin.setAccountCreationFee(accountCreationFee)
+        storageAdmin.setMinimumStorageReservation(minimumStorageReservation)
+    }
+}
+`
+
+const setupStorageForServiceAccountsTemplate = `
+import FlowServiceAccount from 0x%s
+import FlowStorageFees from 0x%s
+import FungibleToken from 0x%s
+import FlowToken from 0x%s
+
+// This transaction sets up storage on any auth accounts that were created before the storage fees.
+// This is used during bootstrapping a local environment 
+transaction() {
+    prepare(service: AuthAccount, fungibleToken: AuthAccount, flowToken: AuthAccount, feeContract: AuthAccount) {
+        let authAccounts = [service, fungibleToken, flowToken, feeContract]
+
+        // take all the funds from the service account
+        let tokenVault = service.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault) ?? panic("Unable to borrow reference to the default token vault")
```

Comment:
Maybe add a line break

```suggestion
        let tokenVault = service.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault) 
            ?? panic("Unable to borrow reference to the default token vault")
```